signup/templates/signup/signup.html

```diff
@@ -10,6 +10,14 @@
 <html>
 <head>
 	<title> Sign Up </title>
+    <style>
```

Comment:
@medhach Could you move the css to its own .css file? You can do this in the next PR.